WindsorBody/Allclean.pre

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+rm -r WindsorBody_OF-v2206_incomp_AutoCFD3g1_test01 slurm-*.out images/*.png images/*.pdf
```

Comment:
In general I would keep the folder names a bit more generic, thus WindrosBody_OF .. wont be in the file structure any more.